normalize.js

```diff
@@ -114,7 +114,7 @@ define(['require', 'module'], function(require, module) {
     fromBase = removeDoubleSlashes(fromBase);
     toBase = removeDoubleSlashes(toBase);
 
-    var urlRegEx = /@import\s*("([^"]*)"|'([^']*)')|url\s*\(\s*(\s*"([^"]*)"|'([^']*)'|[^\)]*\s*)\s*\)/ig;
+    var urlRegEx = /@import\s*((?:"([^"]*)")|(?:'([^']*)'))|url\s*\(\s*(\s*(?:"([^"]*)")|(?:'([^']*)')|[^\)]*\s*)\s*\)/ig;
```

Comment:
The main difference is I'm referencing the capture groups to make sure it's well-formed. So if `(["'])` matches a quotation mark, it looks for a quotation mark to end the sequence.

Sorry that I didn't see this question until the comment below.